st2api/st2api/controllers/v1/packs.py

```diff
@@ -32,6 +55,128 @@
 
 LOG = logging.getLogger(__name__)
 
+ENTITIES = {
+    'action': (ActionsRegistrar, 'actions'),
+    'trigger': (TriggersRegistrar, 'triggers'),
+    'sensor': (SensorsRegistrar, 'sensors'),
+    'rule': (RulesRegistrar, 'rules'),
+    'alias': (AliasesRegistrar, 'aliases'),
+    'policy': (PolicyRegistrar, 'policy'),
+    'config': (ConfigsRegistrar, 'config')
+}
+
+
+class PackInitController(ActionExecutionsControllerMixin, RestController):
+
+    @jsexpose(body_cls=PackInitRequestAPI, status_code=http_client.ACCEPTED)
+    def post(self, args):
+        parameters = vars(args)
+
+        new_liveaction_api = LiveActionCreateAPI(action='packs.create',
+                                                 parameters=parameters,
+                                                 user=None)
```

Comment:
Can we please correctly carry the user context over if possible?

This way RBAC, {{user.}}, will work as expected (I know it's not used right now in that action, but still).